src/solvers/solver_hardness.cpp

```diff
@@ -151,6 +163,16 @@ void solver_hardnesst::produce_report()
       sat_hardness_json["Variables"] =
         json_numbert{std::to_string(hardness.variables.size())};
 
+      json_arrayt sat_hardness_clause_set_json;
+      for(auto const &clause : hardness.clause_set)
+      {
+        json_arrayt clause_json;
+        for(auto const &lit : clause)
+          clause_json.push_back(json_numbert{std::to_string(lit)});
```

Comment:
`json_numbert` takes a string?

Reply:
Yes, takes a string but the output is formatted json number.